src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components/macro';
+
+export const ButtonStyle = styled.button`
```

Comment:
This is not a criticism. What you have is fine. But did you know that you can use the following:

```js
const ButtonStyle = styled('button')`
...
`
```

Reply:
But as your list of components grows (not right now, but in the future), consider pushing all single-file styled components into a subdirectory `/components/styled` for clarity. 

This is not a problem right now, but it might help down the road?

Reply:
just practice doing a bit of styled component. If this grows, I add it later on

Reply:
Rock on